### DISCUSSION TITLE
the boot order

### DISCUSSION THREAD

**Discussion by ghwswywps** [DavidS95/Smokeless_UMAF#4] [Q&A] [answered]
can i change boot order using this tools? 
plz🙏🏻

**Comment by DavidS95** [comment, marked as answer]
Possibly, this just allows for hidden options to be shown, in my case boot order options do appear but thats on a case by case basis. If it does not show up that is not something I can fix as the bios simply does not have that as an option. However more often than not there will be that option there so go mess around in the folders and see if u can spot it, it should be one of the root/top folders.
